Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+FROM alpine:3.13.4
+
+ARG ARCH=amd64
+
+ARG KUBERNETES_VERSION=1.17.3
```

Comment:
Is that kubernentes version related to che-operator somehow ?

Reply:
No, just a version. Updated to the latest stable